packages/components/src/internal/components/assay/PlatePropertiesPanel.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, memo, useMemo } from 'react';
+import Formsy from 'formsy-react';
+
+import { ExtendedMap } from '../../../public/ExtendedMap';
+import { QueryColumn } from '../../../public/QueryColumn';
+
+import { QueryFormInputs } from '../forms/QueryFormInputs';
+
+import { getContainerFilterForLookups } from '../../query/api';
+
+import { AssayPropertiesPanelProps } from './models';
+
+export const PlatePropertiesPanel: FC<AssayPropertiesPanelProps> = memo(({ model, onChange, operation }) => {
+    // FIXME: Update the AssayWizardModel to use ExtendedMap for plateColumns so we don't need to do this conversion.
```

Comment:
I assume this is for a future PR?

Reply:
Yea, copied over from`RunPropertiesPanel`.